client/lib/wp/browser.js

```diff
@@ -59,6 +60,8 @@ if ( 'development' === config( 'env' ) ) {
 // Inject localization helpers to `wpcom` instance
 wpcom = injectLocalization( wpcom );
 
+injectGuestSandboxTicketHandler( wpcom );
```

Comment:
What about keeping the same signature than `injectLocalization()` and similar functions that updates `wpcom`?

```
wpcom = injectGuestSandboxTicketHandler( wpcom );
```

On the other hand, returning an instance like this makes me think that this instance is different from the one given in parameter, which is not the case looking at this function.

Just curious about your thoughts.